numba/cuda/dispatcher.py

```diff
@@ -482,27 +482,24 @@ def __call__(self, *args):
         if self.ntasks == 0:
             return
 
-        if self.dispatcher.specialized:
-            specialized = self.dispatcher
-        else:
-            specialized = self.dispatcher.specialize(*args)
-        blockdim = self._compute_thread_per_block(specialized)
+        blockdim = self._compute_thread_per_block(args)
+        griddim = (self.ntasks + blockdim - 1) // blockdim
+        self.dispatcher[griddim, blockdim, self.stream, self.sharedmem](*args)
         griddim = (self.ntasks + blockdim - 1) // blockdim
```

Comment:
This seems superfluous now - if I'm reading the diff right it recomputes `griddim` just before returning?